js/app.js

```diff
@@ -0,0 +1,107 @@
+console.log('yo')
+// combine objects, arrays, and functions
+const curbArray = [
+	characters: {
+		name1: 'jeff',
+		name2: 'larry',
+		name3: 'susie',
+		name4: 'cheryl',
+		name5: 'funkhouser'
+	}
+]
+console.log(curbArray.name5)
```

Comment:
having made the correction i indicated, this line would need to read `console.log(curbArray[0].name5)` to not produce an error

Reply:
-0.33